packages/is-shallow-equal/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## 3.0.0 (2020-11-12)
```

Comment:
Release tool will add a version number and date. It's all automated based on type of sections listed.